staging/src/k8s.io/kube-aggregator/pkg/apiserver/handler_apis.go

```diff
@@ -75,11 +59,6 @@ var discoveryGroup = metav1.APIGroup{
 }
 
 func (r *apisHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
-	// if the URL is for OUR api group, serve it normally
-	if strings.HasPrefix(req.URL.Path+"/", "/apis/"+apiregistrationapi.GroupName+"/") {
-		r.delegate.ServeHTTP(w, req)
-		return
-	}
 	// don't handle URLs that aren't /apis
 	if req.URL.Path != "/apis" && req.URL.Path != "/apis/" {
 		r.delegate.ServeHTTP(w, req)
```

Comment:
unrelated, but I just noticed we lost `content-type: application/json` on the following URLs:
* `https://<server>/apis`
* `https://<server>/apis/`
* `https://<server>/apis/extensions`
* `https://<server>/apis/extensions/`

Oddly, the same urls against the unsecured port still return `content-type: application/json`. I didn't expect the handler (once you got through the outer filters) to be different.